test/cdn-bundle/test/cdnBundle.test.ts

```diff
@@ -8,18 +8,24 @@ test.describe('CDN bundle', () => {
    * It does this by checking that a webpage which imports this bundle is able to create and use a Spaces client.
    */
   test('browser can import and use the CDN bundle', async ({ page }) => {
+    let resolvePageResult: (value: PromiseLike<string> | string) => void;
+    let rejectPageResult: (reason: any) => void;
+
     const pageResultPromise = new Promise<string>((resolve, reject) => {
-      page.exposeFunction('onResult', (error: Error | null) => {
-        if (error) {
-          reject(error);
-        } else {
-          resolve('Success');
-        }
-      });
+      resolvePageResult = resolve;
+      rejectPageResult = reject;
+    });
 
-      page.exposeFunction('createSandboxAblyAPIKey', createSandboxAblyAPIKey);
+    await page.exposeFunction('onResult', (error: Error | null) => {
```

Comment:
Updated playwright is not as forgiving as previous version, it requires awaiting `page.exposeFunction` calls